docs/observability/how_to_guides/tracing/output_detailed_logs.mdx

```diff
@@ -0,0 +1,48 @@
+import {
+  CodeTabs,
+  PythonBlock,
+  TypeScriptBlock,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to print out logs from the langsmith python package
+
+The LangSmith package uses python's built in [`logging`](https://docs.python.org/3/library/logging.html) mechanism to output logs
+about its behavior to standard output.
+
+## Ensure logging is configured
+
+If logging is not currently configured for your python environment, you'll need to explicitly turn it on as follows:
```

Comment:
whats this needed for again? is this needed if you do the langsmith-specific bit below?

Reply:
jupyter notebooks by default print errors to std err so they don't show up in the cell output. 

I figure customers will run into this... how would you recommend saying this more explicitly?